model-mesh/runtimes/triton-2.x.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: triton-2.x
+  labels:
+    name: modelmesh-serving-triton-2.x-SR
+  annotations:
+    maxLoadingConcurrency: "2"
+spec:
+  supportedModelTypes:
+    - name: tensorflow
+      version: "1" # 1.15.4
+    - name: tensorflow
+      version: "2" # 2.3.1
+    - name: tensorrt
+      version: "7" # 7.2.1
+    - name: pytorch
+      version: "1" # 1.8.0a0+17f8c32
+    - name: onnx
+      version: "1" # 1.5.3
+
+  grpcEndpoint: "port:8085"
+  grpcDataEndpoint: "port:8001"
+
+  containers:
```

Comment:
Can we have multiple pods for this guy? Is there a way to specify metrics exposing? Or does that other controller metrics endpoint have everything?